tests/user.test.js

```
@@ -25,7 +25,7 @@ describe('User', () => {
     try {
       await sequelize.sync({ force: true });
     } catch (err) {
-      console.log(err);
+      // Error occurred
     }
```

This `try...catch` block silently ignores any errors that might occur during `sequelize.sync`. If the database synchronization fails, the tests will continue to run and likely fail in confusing ways. It's better to let the error propagate, which will cause the entire test suite to fail immediately, making the root cause of the problem much clearer. 

Consider removing the `try...catch` block and letting Jest handle any setup failures.